translations/es.ts

```diff
@@ -2349,13 +2349,13 @@ Si no estás seguro, selecciona &quot;No&quot; para evitar comunicaciones usando
     <message>
         <location filename="../src/widget/widget.cpp" line="497"/>
         <source>Toxcore failed to start, the application will terminate after you close this message.</source>
-        <translation>toxcore falló al iniciar, el programa terminará al cerrar este mensaje.</translation>
+        <translation>Toxcore falló al iniciar, el programa terminará al cerrar este mensaje.</translation>
```

Comment:
@zetok: thanks. 
Could you also please fix the English source string in qtox and the text in https://github.com/irungentoo/toxcore/blob/master/INSTALL.md 
Thank you.
(I better stay in the trnaslation only)